src/API/Authentication.php

```diff
@@ -212,47 +182,43 @@ public function email_code_passwordless_verify($email, $code, $scope = 'openid')
 
   }
 
-  public function userinfo($access_token){
-
-    return $this->apiClient->get()
-      ->userinfo()
+  /**
+   * @deprecated Use `login` instead. Use only for passwordless verify
+   */
+  protected function authorize_with_ro($username, $password, $scope = 'openid', $connection = null, $id_token = null, $device = null){
```

Comment:
I wont remove the deprecation notice. they should move to use /oauth/token instead of /ro for authN/authZ